terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Should this lock file be gitignored?

Reply:
This is locking the provider versions, it's not locking state etc - I'd argue it should remain so that my deployment runs with the same provider versions yours or a CI build might

Reply:
Sounds good, thanks.